inspire-tree.d.ts

```diff
@@ -278,6 +279,31 @@ export class TreeNode {
     constructor(tree: InspireTree, source: any|TreeNode);
     constructor(tree: InspireTree);
     text: string;
+    itree?: {
```

Comment:
I believe this entire block is already defined above as `NodeConfig`, you could likely re-use that here instead.

Reply:
Oh sweet, wasn't aware of the ability to reference subtree properties. Implemented now 👍